### PR TITLE
SAK-45680 Add tablesorter styling to tool links table

### PR DESCRIPTION
**Pull request by davidpbauer**

### CODE REVIEW

**Comment by davidpbauer** [comment]
@csev This manually adds the classes needed to match the blue tablesorter theme on the "installed tools" tab.

File: basiclti/basiclti-tool/src/webapp/vm/lti_tool_site.vm [hunk 1]
```diff
@@ -130,9 +130,9 @@ ${includeLatestJQuery}
             </select>
         </div>
 		#end
-		<table id="tools_site" class="tablesorter" cellspacing="0" summary ="$tlang.getString("content.table")">
+		<table id="tools_site" class="tablesorter tablesorter-blue hasFilters" cellspacing="0" summary ="$tlang.getString("content.table")">
```
Comment (ottenhoff) [inline basiclti/basiclti-tool/src/webapp/vm/lti_tool_site.vm:133]: my only argument here is that the blue is a bit of a different design than everywhere else.... would rather have this neutral colors

Comment (davidpbauer) [reply basiclti/basiclti-tool/src/webapp/vm/lti_tool_site.vm:133]: Agreed. "Blue" was already added as the theme in this file and for the lti_tool_system.vm table. I think we'd want to switch both to a new theme if we change it here.
`<script type="text/javascript">includeWebjarLibrary('jquery.tablesorter/2.27.7/dist/css/theme.blue.min.css');</script>`

Comment (csev) [reply basiclti/basiclti-tool/src/webapp/vm/lti_tool_site.vm:133]: I say we merge this - so as to undo the regression - (i.e. restore it to how it was when it worked) - and then clean up colors later.

Comment (ern) [reply basiclti/basiclti-tool/src/webapp/vm/lti_tool_site.vm:133]: New JIRA will be opened to address neutral theme of the table, thanks @profmikegreene

**Review by csev — APPROVED** [review]

**Pull request merged**